packages/desktop/views/dashboard/collectibles/views/CollectiblesGalleryView.svelte

```diff
@@ -1,11 +1,21 @@
 <script lang="ts">
     import { localize } from '@core/i18n'
-    import { nftSearchTerm, ownedNfts, queriedNfts } from '@core/nfts/stores'
+    import { nftSearchTerm, ownedNfts } from '@core/nfts/stores'
     import { NftGallery, SearchInput } from '@ui'
     import { Button, IconName, Text, Pill } from '@bloomwalletio/ui'
     import { PopupId, openPopup } from '@desktop/auxiliary/popup'
     import features from '@features/features'
     import { CollectiblesListMenu, EmptyListPlaceholder } from '@components'
+    import { Filter } from '@components/filter'
+    import { nftFilter } from '@core/nfts/stores/nft-filter.store'
```

Comment:
Update imports:
- Add `nftFilter` to l.3 import
- Don't import from top level folder, as this can potentially cause circular dependencies.